### PR TITLE
Readd validate inventory and display plan, remove os_images and release_images and set default installer to agent based. 

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

**Review by arjuhe — APPROVED** [review]
LGTM

File: hack/pip-requirments.txt
Comment (arjuhe) [inline hack/pip-requirments.txt:1]: Just add these direct to the requirements.txt. It would make the install process a little easier.

**Review by arjuhe — APPROVED** [review]
LGTM.

**Review by arjuhe — APPROVED** [review]
LGTM

File: hack/pip-requirments.txt
Comment (nocturnalastro) [reply hack/pip-requirments.txt:1]: I have removed the requirements file and instead I've just put the instructions in the README

File: docs/inventory.md [hunk 1]
```diff
@@ -727,12 +748,53 @@ all:
...
 
-
-# Operators
+## Operators
 
 It is possible to install a few operators as part of the cluster installtion. These operators are Local Storage Operator (`install_lso: True`), Open Data Fabric (`install_odf: True`) and Openshift Virtualization (`install_cnv: True`)
+
+## Day 2 nodes
+
+Day 2 nodes are added to an existing cluster. The reason why the installation of day 2 nodes is built into the main path of our automation, is that for assisted installer day 2 nodes can be on a different L2 network which the main flow does not allow.
```
Comment (arjuhe) [inline docs/inventory.md:767]: sentence per line

[edited to correct spelling]

File: hack/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# generate_os_release_images.py
+
+## Requriments
+
+```shell
+pip install semver beautifulsoup4
+```
+
+## Useage
```
Comment (arjuhe) [inline hack/README.md:9]: spelling

**Review by arjuhe — APPROVED** [review]
Thanks LGTM

**Pull request merged**